Signal/src/ViewControllers/OWSSoundSettingsViewController.m

```diff
@@ -74,13 +74,25 @@ - (void)updateTableContents
     for (NSNumber *nsValue in allSounds) {
         OWSSound sound = (OWSSound)nsValue.intValue;
         OWSTableItem *item;
+
+        NSString *soundLabelText = ^{
+            NSString *baseName = [OWSSounds displayNameForSound:sound];
+            if (sound == OWSSound_Note) {
+                NSString *noteStringFormat = NSLocalizedString(@"SETTINGS_AUDIO_DEFAULT_TONE_LABEL_FORMAT",
+                    @"Format string for the default 'Note' sound. Embeds the system {{sound name}}.");
+                return [NSString stringWithFormat:noteStringFormat, baseName];
```

Comment:
This is fine, but lately, I've been doing ("string a" + [" "] + "string b") using the rtl-safe string concatenation so that we don't have to count on translator to get the RTL ordering correct, since they have less context than we do.

Reply:
Good point.